collections/elixirPosts.js

```diff
@@ -0,0 +1,11 @@
+const livePosts = require("./posts");
+
+module.exports = collection => {
+  const allPosts = livePosts(collection);
+  return [
+    ...allPosts.filter(
+      post =>
+        Array.isArray(post.data.tags) && post.data.tags.some(t => t.match(/^(elixir|phoenix)/))
+    ),
+  ];
```

Comment:
The spread operator seems to be redundant.

```suggestion
  return allPosts.filter(
      post =>
        Array.isArray(post.data.tags) && post.data.tags.some(t => t.match(/^(elixir|phoenix)/))
    ),
```

It would probably make sense to make the regex case-insensitive.

```suggestion
  return [
    ...allPosts.filter(
      post =>
        Array.isArray(post.data.tags) && post.data.tags.some(t => t.match(/^(elixir|phoenix)/i))
    ),
  ];
```

Lastly, I'm not exactly sure why the "starts with" pattern is needed here (`^`) but it seems fine.

Reply:
right – fixed 👍